assets/chezmoi.io/docs/reference/templates/doppler/doppler.md

```diff
@@ -0,0 +1,15 @@
+# `doppler` *key* [*project* [*config*]]
+
+`doppler` returns the plaintext secret from [Doppler](https://www.doppler.com) using `doppler
+secrets get --plain`.
+
+If any of *project*, or *config* are empty or
+omitted, then chezmoi will use the value from the
+`doppler.project`, and
+`doppler.config` config variables if they are set and not empty.
```

Comment:
I mentioned the wording on `dopplerProjectJson`, but the same pattern would be used here.